src/main/java/com/keshavarzi/airplanestats/exception/register/EmailExistException.java

```diff
@@ -0,0 +1,16 @@
+package com.keshavarzi.airplanestats.exception.register;
+
+public class EmailExistException extends Exception {
+
+    /**
+     * Constructs a new exception with the specified detail message.  The
+     * cause is not initialized, and may subsequently be initialized by
+     * a call to {@link #initCause}.
+     *
+     * @param message the detail message. The detail message is saved for
+     *                later retrieval by the {@link #getMessage()} method.
+     */
+    public EmailExistException(String message) {
```

Comment:
Nit: For clarity I'd rename this to `EmailAlreadyExistsException` which indicates that it wasn't supposed to exist